### ISSUE TITLE
[Feature Request]: Add support for native images

### ISSUE THREAD

**Issue by SamTV12345** [Frooodle/Stirling-PDF#264]
I would love to have native images that are around 30-50% faster than plain old Java applications and take up less compute power and memory.

**Comment by Frooodle** [comment]
Didn't know this was a thing, looks good I'll try it out

**Comment by SamTV12345** [comment]
Yes. It's really cool. Bell Soft gave a very interesting talk on [this](https://github.com/jugsaar/jugsaar-64-perf-test-for-scaling/blob/main/K8s_en_Jugs_slides.pdf).
